### PR TITLE
Update basic.mdx ( Add Some important Selectors tags )

### PR DESCRIPTION
**Pull request by Fuzaildev**
Update Some Important Selectors Tags that are used in CSS.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: content/batch/learn/css/basic.mdx [hunk 1]
```diff
@@ -121,6 +121,38 @@ h1 {
   color: green;
 }
 
+Add More important Selectors tags that really used.
```
Comment (Vishal-raj-1) [inline content/batch/learn/css/basic.mdx:124]: you don't need to add these two lines so remove it

File: content/batch/learn/css/basic.mdx [hunk 2]
```diff
@@ -121,6 +121,38 @@ h1 {
...
+
+/* Universal Selector (ALL Elements margin and padding are 0; ) */
+
+{
+margin: 0;
+padding: 0;
+}
+/* Pseudo Selector */
+
+/* If you visited the website then you can change the color */
```
Comment (Vishal-raj-1) [inline content/batch/learn/css/basic.mdx:146]: Pseudo Selector is discussed in link tag, so you can remove it from here

**Review by Vishal-raj-1 — CHANGES_REQUESTED** [review]
Fix these changes

File: content/batch/learn/css/basic.mdx (hunk 2 above)
Comment (Fuzaildev) [reply content/batch/learn/css/basic.mdx]: Done

**Pull request merged**